catkit2/services/oceanoptics_spectrometer_sim/oceanoptics_spectrometer_sim.py

```diff
@@ -0,0 +1,156 @@
+'''
+This module contains a service for simulation Ocean Optics Spectrometers.
+'''
+import numpy as np
+from catkit2.testbed.service import Service
+
+
+class OceanOpticsSpectrometerSim(Service):
+    '''
+    Service for Ocean Optics Spectrometer.
+
+    This service is a wrapper around the python seabreeze package.
+    It provides a simple interface to control the spectrometer and acquire spectra.
+
+    Attributes
+    ----------
+    spectrometer : Spectrometer
+        The spectrometer to control.
+    serial_number : str
+        The serial number of the spectrometer.
+    model : str
+        The model of the spectrometer.
+    pixels_number : int
+        number of pix in the spectrum.
```

Comment:
You probably copied this too early ^^ - pleas write out "pixels" here. And capitalize "Number".